### PR TITLE
[CI] Config pr and issue assignee

### PR DESCRIPTION
**Pull request by AllyW**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
CI

File: .github/policies/resourceManagement.yml [hunk 1]
```diff
@@ -3215,6 +3215,9 @@ configuration:
           label: Azure Data Explorer
       - addLabel:
           label: Service Attention
+      - assignTo:
+          users:
+            - evelyn-ys
```
Comment (wangzelin007) [inline .github/policies/resourceManagement.yml:3220]:
```suggestion
          - evelyn-ys

```

File: .github/policies/resourceManagement.yml [hunk 2]
```diff
@@ -3859,6 +3862,53 @@ configuration:
...
+        - addLabel:
+            label: Logic App
+        - requestReview:
+            reviewer: zhoxing-ms
+        - requestReview:
+            reviewer: ReaNAiveD
+        - assignTo:
+            users:
+              - zhoxing-ms
+              - ReaNAiveD
```
Comment (wangzelin007) [inline .github/policies/resourceManagement.yml:3910]:
```suggestion
            - zhoxing-ms
            - ReaNAiveD

```

File: .github/policies/resourceManagement.yml [hunk 3]
```diff
@@ -4665,6 +4716,10 @@ configuration:
           label: extension/grafana
       - addLabel:
           label: Service Attention
+      - assignTo:
+          users:
+            - AllyW
+            - kairu-ms
```
Comment (wangzelin007) [inline .github/policies/resourceManagement.yml:4722]:
```suggestion
          - AllyW
          - kairu-ms

```

File: .github/policies/resourceManagement.yml [hunk 4]
```diff
@@ -5348,7 +5410,42 @@ configuration:
           label: Attestation
       - addLabel:
           label: Service Attention
+      - assignTo:
+          users:
+            - ReaNAiveD
```
Comment (wangzelin007) [inline .github/policies/resourceManagement.yml:5415]:
```suggestion
          - ReaNAiveD

```

File: .github/policies/resourceManagement.yml [hunk 5]
```diff
@@ -5348,7 +5410,42 @@ configuration:
...
+          action: Opened
+      - or:
+        - titleContains:
+            pattern: '[Aa]z attestation'
+            isRegex: True
+        - bodyContains:
+            pattern: '[Aa]z attestation'
+            isRegex: True
+        - titleContains:
+            pattern: '\b([A]ttestation)\b'
```
Comment (wangzelin007) [inline .github/policies/resourceManagement.yml:5429]:
```suggestion
            pattern: '\b([Aa]ttestation)\b'

```

File: .github/policies/resourceManagement.yml [hunk 6]
```diff
@@ -5348,7 +5410,42 @@ configuration:
...
+            isRegex: True
+        - bodyContains:
+            pattern: '\b([A]ttestation)\b'
```
Comment (wangzelin007) [inline .github/policies/resourceManagement.yml:5432]:
```suggestion
            pattern: '\b([Aa]ttestation)\b'

```

File: .github/policies/resourceManagement.yml [hunk 7]
```diff
@@ -5927,5 +6088,255 @@ configuration:
...
+      - payloadType: Issues
+      - or:
+        - titleContains:
+            pattern: '[Aa]z powerbi'
+            isRegex: True
+        - bodyContains:
+            pattern: '[Aa]z powerbi'
+            isRegex: True
+        - titleContains:
+            pattern: '\b([P]owerbi)\b'
```
Comment (wangzelin007) [inline .github/policies/resourceManagement.yml:6288]:
```suggestion
            pattern: '\b([Pp]owerbi)\b'

```

File: .github/policies/resourceManagement.yml [hunk 8]
```diff
@@ -5927,5 +6088,255 @@ configuration:
...
+            isRegex: True
+        - bodyContains:
+            pattern: '\b([P]owerbi)\b'
```
Comment (wangzelin007) [inline .github/policies/resourceManagement.yml:6291]:
```suggestion
            pattern: '\b([Pp]owerbi)\b'

```

File: .github/policies/resourceManagement.yml [hunk 9]
```diff
@@ -3859,6 +3862,53 @@ configuration:
...
+                action: Opened
+      then:
+        - addLabel:
+            label: Logic App
+        - addLabel:
+            label: Auto-Assign
+        - assignTo:
+            users:
+              - ReaNAiveD
+              - zhoxing-ms
```
Comment (wangzelin007) [inline .github/policies/resourceManagement.yml:3885]:
```suggestion
            - ReaNAiveD
            - zhoxing-ms

```

File: .github/policies/resourceManagement.yml [hunk 10]
```diff
@@ -5927,5 +6095,257 @@ configuration:
...
+        - bodyContains:
+            pattern: '[Aa]z powerbi'
+            isRegex: True
+        - titleContains:
+            pattern: '\b([Pp]owerbi)\b'
+
+            isRegex: True
+        - bodyContains:
+            pattern: '\b([Pp]owerbi)\b'
+
```
Comment (AllyW) [inline .github/policies/resourceManagement.yml:6300]:
```suggestion
```

File: .github/policies/resourceManagement.yml [hunk 11]
```diff
@@ -3859,6 +3863,55 @@ configuration:
...
+      then:
+        - addLabel:
+            label: Logic App
+        - addLabel:
+            label: Auto-Assign
+        - assignTo:
+            users:
+            - ReaNAiveD
+            - zhoxing-ms
+
```
Comment (AllyW) [inline .github/policies/resourceManagement.yml:3887]:
```suggestion

```

File: .github/policies/resourceManagement.yml [hunk 12]
```diff
@@ -3859,6 +3863,54 @@ configuration:
...
+            label: Logic App
+        - requestReview:
+            reviewer: zhoxing-ms
+        - requestReview:
+            reviewer: ReaNAiveD
+        - assignTo:
+            users:
+            - zhoxing-ms
+            - ReaNAiveD
+
```
Comment (AllyW) [inline .github/policies/resourceManagement.yml:3912]:
```suggestion

```

File: .github/policies/resourceManagement.yml [hunk 13]
```diff
@@ -5348,7 +5413,45 @@ configuration:
           label: Attestation
       - addLabel:
           label: Service Attention
+      - assignTo:
+          users:
+          - ReaNAiveD
+
```
Comment (AllyW) [inline .github/policies/resourceManagement.yml:5419]:
```suggestion

```

File: .github/policies/resourceManagement.yml [hunk 14]
```diff
@@ -5348,7 +5413,45 @@ configuration:
...
+      - or:
+        - titleContains:
+            pattern: '[Aa]z attestation'
+            isRegex: True
+        - bodyContains:
+            pattern: '[Aa]z attestation'
+            isRegex: True
+        - titleContains:
+            pattern: '\b([Aa]ttestation)\b'
+
```
Comment (AllyW) [inline .github/policies/resourceManagement.yml:5434]:
```suggestion

```

File: .github/policies/resourceManagement.yml [hunk 15]
```diff
@@ -5348,7 +5413,45 @@ configuration:
...
+            isRegex: True
+        - bodyContains:
+            pattern: '\b([Aa]ttestation)\b'
+
```
Comment (AllyW) [inline .github/policies/resourceManagement.yml:5438]:
```suggestion

```

File: .github/policies/resourceManagement.yml [hunk 16]
```diff
@@ -5927,5 +6094,256 @@ configuration:
...
+          action: Opened
+      - or:
+        - titleContains:
+            pattern: '[Aa]z storagesync'
+            isRegex: True
+        - bodyContains:
+            pattern: '[Aa]z storagesync'
+            isRegex: True
+        - titleContains:
+            pattern: '\b([Ss]storagesync)\b'
```
Comment (AllyW) [inline .github/policies/resourceManagement.yml:6199]:
```suggestion
            pattern: '\b([Ss]toragesync)\b'

```

File: .github/policies/resourceManagement.yml [hunk 17]
```diff
@@ -5927,5 +6094,256 @@ configuration:
...
+            isRegex: True
+        - bodyContains:
+            pattern: '\b([Ss]storagesync)\b'
```
Comment (AllyW) [inline .github/policies/resourceManagement.yml:6202]:
```suggestion
            pattern: '\b([Ss]toragesync)\b'

```

File: .github/policies/resourceManagement.yml [hunk 18]
```diff
@@ -5927,5 +6094,256 @@ configuration:
...
+      - or:
+        - titleContains:
+            pattern: '[Aa]z powerbi'
+            isRegex: True
+        - bodyContains:
+            pattern: '[Aa]z powerbi'
+            isRegex: True
+        - titleContains:
+            pattern: '\b([Pp]owerbi)\b'
+
```
Comment (AllyW) [inline .github/policies/resourceManagement.yml:6295]:
```suggestion

```

File: .github/policies/resourceManagement.yml [hunk 19]
```diff
@@ -5927,5 +6090,257 @@ configuration:
...
+      - or:
+        - titleContains:
+            pattern: '[Aa]z storagesync'
+            isRegex: True
+        - bodyContains:
+            pattern: '[Aa]z storagesync'
+            isRegex: True
+        - titleContains:
+            pattern: '\b([Ss]toragesync)\b'
+
```
Comment (AllyW) [inline .github/policies/resourceManagement.yml:6196]:
```suggestion

```

File: .github/policies/resourceManagement.yml [hunk 20]
```diff
@@ -5927,5 +6090,257 @@ configuration:
...
+            isRegex: True
+        - bodyContains:
+            pattern: '\b([Ss]toragesync)\b'
+
```
Comment (AllyW) [inline .github/policies/resourceManagement.yml:6200]:
```suggestion

```

File: .github/policies/resourceManagement.yml [hunk 21]
```diff
@@ -3215,6 +3215,10 @@ configuration:
           label: Azure Data Explorer
       - addLabel:
           label: Service Attention
+      - assignTo:
+          users:
+          - evelyn-ys
+
```
Comment (AllyW) [inline .github/policies/resourceManagement.yml:3221]:
```suggestion

```

**Review by wangzelin007 — APPROVED** [review]

**Pull request merged**